infrastructure-playbooks/recover-osds-after-ssd-journal-failure.yml

```diff
@@ -0,0 +1,126 @@
+---
+# This playbook use to recover Ceph OSDs after ssd journal failure.
+# You will also realise that it’s really simple to bring your 
+# OSDs back to life after replacing your faulty SSD with a new one.
+#
+# You should define `dev_ssds` variable for host which change ssds after
+# failture. 
+# 
+# For example in host_vars/hostname1.yml
+#
+# dev_ssds:
+# - device_name: sdd
```

Comment:
people might want to use device path like this: `/dev/disk/by-`, so please do not assume `/dev/sdb`.